lib/application/presentation/detail_screen.dart

```diff
@@ -1,25 +1,294 @@
+                          child: Row(
+                            mainAxisAlignment: MainAxisAlignment.spaceBetween,
+                            crossAxisAlignment: CrossAxisAlignment.start,
+                            children: [
+                              Wrap(
+                                spacing: 5,
+                                children: [
+                                  Icon(
+                                    Icons.circle,
+                                    color: widget.scheduleItem.category ==
+                                            ScheduleItemCategory.tech
+                                        ? MyColors.colorF44336
+                                        : widget.scheduleItem.category ==
+                                                ScheduleItemCategory.lead
+                                            ? MyColors.color9B9A9B
+                                            : widget.scheduleItem.category ==
+                                                    ScheduleItemCategory.ops
+                                                ? MyColors.color251F5D
+                                                : MyColors.color000000,
+                                    size: 14,
+                                  ),
+                                  Text(
+                                    "${widget.scheduleItem.startTime.hour.toString()}:${widget.scheduleItem.startTime.minute.toString().padLeft(2, '0')} - ${widget.scheduleItem.endTime.hour.toString()}:${widget.scheduleItem.endTime.minute.toString().padLeft(2, '0')}",
+                                    style:
+                                        Theme.of(context).textTheme.labelMedium,
+                                  ),
+                                ],
+                              ),
+                              Wrap(
+                                spacing: 5,
+                                children: [
+                                  const Icon(
+                                    Icons.location_pin,
+                                    size: 14,
+                                    color: MyColors.color9B9A9B,
+                                  ),
+                                  Text(
+                                    widget.scheduleItem.hall.toUpperCase(),
+                                    style:
+                                        Theme.of(context).textTheme.labelMedium,
+                                  ),
+                                ],
+                              ),
+                            ],
+                          ))),
+                  Align(
+                      alignment: Alignment.centerLeft,
+                      child: Padding(
+                          padding: const EdgeInsets.only(
+                              left: 40, right: 40, top: 20),
+                          child: RichText(
+                            text: TextSpan(
+                              text:
```

Comment:
Što nebi trebao naziv kategorije biti obojan?

Reply:
Trebao bi samo sto nisam siguran odkud boja da se uzme, vjv cu napravit nes slicno kao i sa onim kruzicem